packages/core-data/src/selectors.ts

```diff
@@ -140,6 +140,33 @@ const EMPTY_OBJECT = {};
  * @param state Data state.
  * @param url   URL the preview would be for.
  *
+ * @example
+ * ```js
+ * import { store as coreDataStore } from '@wordpress/core-data';
+ * import { useSelect } from '@wordpress/data';
+ *
+ * const ExampleComponent = () => {
+ *     const { embedPreview, isRequestingEmbedPreview } = useSelect(
+ *         ( select ) => {
+ *             return {
+ *                 embedPreview: select( coreDataStore ).getEmbedPreview(
+ *                     'https://twitter.com/wordpress'
+ *                 ),
+ *                 isRequestingEmbedPreview: select(
+ *                     coreDataStore
+ *                 ).isRequestingEmbedPreview( 'https://twitter.com/wordpress' ),
+ *             };
+ *         }
+ *     ,[]);
```

Comment:
Nit, code styling guidelines could get improved: 
```suggestion
 *         },
 *         []
 *.     );
```